user-management-oidc.md

```diff
@@ -11,14 +11,44 @@ The OIDC integration in Kubecost is fulfilled via the helm configuration paramet
 
 Please refer to the following references to find out more about how to configure the Helm parameters to suit each OIDC identiy provider integration.
 
-* [Gluu docs](https://gluu.org/docs/gluu-server/4.0/admin-guide/openid-connect/)
-* [Okta docs](https://developer.okta.com/docs/reference/api/oidc/#request-parameters)
+* [Auth0 docs](https://auth0.com/docs/get-started/authentication-and-authorization-flow/add-login-auth-code-flow)
 * [Azure docs](https://learn.microsoft.com/en-us/azure/active-directory/develop/v2-protocols-oidc#send-the-sign-in-request)
+* [Gluu docs](https://gluu.org/docs/gluu-server/4.0/admin-guide/openid-connect/)
+* [Keycloak (see below)](#keycloak-setup)
 * [Google OAuth 2.0 docs](https://developers.google.com/identity/openid-connect/openid-connect#authenticatingtheuser)
-* [Auth0 docs](https://auth0.com/docs/get-started/authentication-and-authorization-flow/add-login-auth-code-flow)
+* [Okta docs](https://developer.okta.com/docs/reference/api/oidc/#request-parameters)
 
 > **Note**: Auth0 does not support Introspection; therefore we can only validate the access token by calling /userinfo within our current remote token validation flow. This will cause the Kubecost UI to not function under an Auth0 integration, as it makes a large number of continuous calls to load the various components on the page and the Auth0 /userinfo endpoint is rate limited. Independent calls against Kubecost endpoints (eg. via cURL or Postman) should still be supported.
 
+## Keycloak setup
+
+1. Create a new [Keycloak Realm](https://www.keycloak.org/getting-started/getting-started-kube#_create_a_realm).
+2. Navigate to "Realm Settings" -> "General" -> "Endpoints" -> "OpenID Endpoint Configuration" -> "Clients".
+3. Click "Create" to add Kubecost to the list of clients. Define a `clientID`. Ensure the "Client Protocol" is set to `openid-connect`.
+4. Click on your newly created client, then go to "Settings".
+   1. Set "Access Type" to `confidential`.
+   2. Set "Valid Redirect URIs" to `http://your-kubecost-address/model/oidc/authorize`.
+   3. Set "Base URL" to `http://your-kubecost-address`.
+
+The [`.Values.oidc`](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/values.yaml) for Keycloak should be as follows:
```

Comment:
Suggest we use a permalink to GitHub files.
ex: https://github.com/kubecost/cost-analyzer-helm-chart/blob/93d3255870766d236628359a969b7691450d0095/cost-analyzer/values.yaml#L194-L202

Reply:
Would apply to the same link near the top of the page as well.

Reply:
@srpomeroy Updated to become permalinks!